### DISCUSSION TITLE
Waybar bluetooth icon and network icon too small

### DISCUSSION THREAD

**Discussion by maharajg** [prasanthrangan/hyprdots#178] [Q&A] [answered]
I've increased the font size for my waybar, everything looks fine except for the bluetooth icon and network icon, which are are very small compared to the others,

![230907_21h15m42s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/27678845/e0515ce8-e149-414c-af8c-5d0d32d38dbe)

Any suggestions?

Thanks

**Comment by prasanthrangan** [comment, marked as answer]
Hi, It should adapt to bar height set in config.ctl.. but you can also hardcode the px size in [here](https://github.com/prasanthrangan/hyprdots/blob/79807bdfdfb26f94d342ed7b5312754ceccee961/Configs/.config/waybar/modules/tray.jsonc#L2)...

**Reply by maharajg** [reply]
Thank you, bar height fixed the size.
